src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -100,6 +101,75 @@ def pileupSizeTask(self):
             self.logger.exception(msg)
         return cmsDict, cusDict
 
+    def partialPileupTask(self, rseExpression='T1_FNAL_Disk|T2_CERN_Disk'):
+        """
+        Execute partial pileup placement according to the following logic:
+
+        - get rucio DIDs (datasets, in CMS name blocks) for our pileup document
+        - get portion of DIDs based on ceil(containerFraction * num_rucio_datasets)
+        - create new container DID in Rucio via Rucio wrapper createContainer API
+          which calls rucio CLI add_container
+        - update MSPileup JSON with
+          - create new custom Name as pileup+extention
+          - add new transition record and update MSPileup document
+        - we call attachDIDs Rucio wrapper API with our set of DIDs
+          - check if rse is mandatory in attachDIDs API, we prefer to not pass it, otherwise
+          we will pass rses from pileup document
+          - create new rules for custom DID using either FNAL disk or T2_CERN sites
```

Comment:
we should not have these storage default. It's better to have either None or '' (empty string), whatever is the default value in Rucio.

Same comment goes for this function signature, please remove those defaults.

Reply:
done